tests/integration/test_github.py

```diff
@@ -259,6 +259,22 @@ def test_get_tag_from_tag_name(self):
     def test_get_tag_from_nonexisting_tag_name(self):
         assert not self.ogr_project.get_tag_from_tag_name("future")
 
+    def test_get_tags(self):
+        tags = self.ogr_project.get_tags()
+
+        names = {f"0.{i}.0" for i in range(1, 10)}
+        names.update({"0.0.1", "0.0.2", "0.0.3", "0.3.1"})
+        assert names == set(map(lambda tag: tag.name, tags))
```

Comment:
Isn't this a bit fragile? We will break this test on each ogr release.

Reply:
should be fixed now

Reply:
Thanks!